package.json

```diff
@@ -2,17 +2,28 @@
   "name": "eslint-scope",
   "description": "ECMAScript scope analyzer for ESLint",
   "homepage": "http://github.com/eslint/eslint-scope",
-  "main": "lib/index.js",
+  "main": "./dist/eslint-scope.cjs",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./lib/index.js",
+      "require": "./dist/eslint-scope.cjs"
+    },
+    "./package.json": "./package.json"
+  },
   "version": "5.1.1",
   "engines": {
-    "node": ">=8.0.0"
+    "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
   },
   "repository": "eslint/eslint-scope",
   "bugs": {
     "url": "https://github.com/eslint/eslint-scope/issues"
   },
   "license": "BSD-2-Clause",
   "scripts": {
+    "prepare": "npm run update-version && npm run build",
```

Comment:
A problem with this is that our release process doesn't commit `lib/version.js`, so after the `6.0.0` release we'll still have `const version = "5.1.1";` in `lib/version.js`.

Reply:
This would then cause the next release (e.g. `6.1.0`) to fail: `npm install` will trigger the `prepare` script, which will overwrite `lib/version.js` with `6.0.0`. Then, [npm version](https://github.com/eslint/eslint-release/blob/0fdda4da083887abfb7b11a3b1ed8e82c4d3d99e/lib/release-ops.js#L383) will throw error "Git working directory not clean".

Reply:
I’ll fix this in a separate PR.